CIScripts/Common/Job.ps1

```diff
@@ -76,7 +77,11 @@ class JobStep {
 
     Print([int] $IndentLevel) {
         $msg = ""
-        1..($IndentLevel) | ForEach-Object{ $msg += " " }
+        1..($IndentLevel) | ForEach-Object {
+            [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+                "", Justification="Issue #804")]
```

Comment:
should be something like "Issue #804 from PSScriptAnalyzer" or just paste the link https://github.com/PowerShell/PSScriptAnalyzer/issues/804